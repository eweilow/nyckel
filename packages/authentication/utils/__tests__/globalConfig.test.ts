import { createGlobalAuthenticationConfig } from "../..";

describe("createGlobalAuthenticationConfig", () => {
  it("should work correctly", () => {
    expect(
      createGlobalAuthenticationConfig(
        "param:clientId",
        "param:clientSecret",
        "http://domain.com",
        "param:audience"
      )
    ).toMatchSnapshot();
  });
});
