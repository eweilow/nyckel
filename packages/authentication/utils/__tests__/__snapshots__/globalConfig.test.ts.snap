// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createGlobalAuthenticationConfig should work correctly 1`] = `
Object {
  "audience": "param:audience",
  "authorizationDomain": "http://domain.com",
  "clientId": "param:clientId",
  "clientSecret": "param:clientSecret",
  "urls": Object {
    "authorization": "http://domain.com/authorize",
    "issuer": "http://domain.com/",
    "jwks": "http://domain.com/.well-known/jwks.json",
    "logout": "http://domain.com/v2/logout",
    "token": "http://domain.com/oauth/token",
    "userinfo": "http://domain.com/userinfo",
  },
}
`;
